packages/editor/src/store/controls.js

```diff
@@ -1,17 +1,34 @@
 /**
  * WordPress dependencies
  */
+import { default as triggerFetch } from '@wordpress/api-fetch';
 import { createRegistryControl } from '@wordpress/data';
 
-/**
- * Dispatches an action.
```

Comment:
We had a perfectly good DocBlock which was abolished?

Reply:
hmm not intentional, maybe an artifact from resolving a merge conflict maybe.

Reply:
Take note that this doc block isn't really accurate, so I suspect this was a merge conflict artifact (and likely the reason I resolved the way I did).  I suspect the doc block was on some other control action in here so to resolve, I'll just add doc blocks for all the actions :)